nvim-telescope/telescope.nvim
# Complete rewrite of README

- Changes
  - New structure:
    - Overview
      - Installation (add examples for three plugin managers)
      - Dependencies (dependencies sounds better then requirements :D < opinionated)
      - Quick Start (to test telescope)
    - Configuration (contains a quick overview of configuration options and mappings)
    - FAQ (How to questions with answers)
    - API (oh boy needs review, and where most of todos is located. @tjdevries help :D)
    - Media (moved all links and videos here)
    - Recipes (some recipes I thought will be make it easier to configure telescope)

  - Strip previews, I feel telescope preview and screenshot should be located in
    gallery.  It might be good to include a signle preview in the introduction,
    but I feel it may constrian the expectations of new users, "This what I
    expect telescope offer", where as having previews only in gallery, "Oh
    telescope can be configured to look in different ways". :D < opinionated

  - Add faq section, any question answered in issues should be written here.
    (future pr I guess.) Add some questions I previously had, but more is needed
    to be added here.

  - Add configuration recipes based on how I configure telescope, @tjdevries
    please consider having the second recipe a builtin function. (perhaps with
    introducing an additional parameter for the dict)

TODO:

  - Fix links
  - Proofreading introduction
